arch/src/main/java/knaufdan/android/arch/mvvm/IAndroidBaseViewModel.kt

```diff
@@ -4,14 +4,10 @@ import android.os.Bundle
 import androidx.lifecycle.LifecycleObserver
 
 interface IAndroidBaseViewModel : LifecycleObserver {
```

Comment:
Delete interface and move method to abstract implementation`AndroidBaseViewModel`

Reply:
done.